tsconfig.json

```diff
@@ -11,8 +11,8 @@
     "strict": true,
     "esModuleInterop": true,
     "declaration": true,
-    "declarationDir": "dist/dts/",
     "emitDeclarationOnly": true,
+    "outDir": "dist",
```

Comment:
when running the build twice I'm getting errors like this one.
(!) [plugin typescript] @rollup/plugin-typescript TS5055: Cannot write file '/..../spv-wallet-js-client/dist/src/index.d.ts' because it would overwrite input file.

I believe that you should exclude dist from inputs for typescript

Reply:
thanks, I will add this in.